cmd/gateway/s3x/s3.proto

```diff
@@ -114,8 +107,8 @@ message ObjectPartInfo {
 }
 
 message MultipartUpload {
-    string bucket = 1;
-    string object = 2;
+    ObjectInfo objectInfo = 1;
     string id = 3;
```

Comment:
this should be `string id = 2` now that the `object` field was removed

Reply:
ok